module/Motion/LineTracing.cpp

```diff
@@ -72,8 +72,8 @@ void LineTracing::logRunning()
   // targetValueと%~のオーバーライド必須
   snprintf(buf, BUF_SIZE,
            "Run \"targetValue\" LineTracing (\"targetValue\": , targetSpeed: %.2f, "
-           "targetBrightness: %d, basePwm: %d, gain: "
+           "targetBrightness: %d, gain: "
            "(%.2f,%.2f,%.2f), isLeftEdge: %s)",
```

Comment:
フォーマットで改行されないなら１行がいいかな
```suggestion
           "targetBrightness: %d, gain: (%.2f,%.2f,%.2f), isLeftEdge: %s)",
```